pants

```diff
@@ -1,14 +1,14 @@
 #!/usr/bin/env bash
-# Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
I do not think we should be changing the copyright year here. Am I missing the reason for this change?

Reply:
I know usually we shouldn't update copyright, but I wanted to make an exception here so that the script does not seem outdated, since users interact so directly with this one.

From what I can tell, the main reason we don't update copyright year is to act as a historical record, right? In Pants code, which users don't directly interact with, I think that's great. Here, that is less of a priority in my opinion.

Can revert back if this is still wrong to make the exception.

Reply:
cc @benjyw who had thoughts on this the last time I remember talking about this^